sql/base/realmd.sql

```diff
@@ -16,30 +16,12 @@
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
 --
--- Table structure for table `realmd_db_version`
```

Comment:
It's an autogenerated file. I guess the exporter changed slightly since the table was last updated in 2013. ;)